### ISSUE TITLE
Improve interface.

### ISSUE THREAD

**Issue by Ayushk4** [NolanoOrg/cformers#25]
- [ ] Allow stopping-generation midway
- [ ] Anti-prompt for conversation models
- [ ] Support returning embeddings and logits instead of token ids.
